osu.Game/Screens/Play/Player.cs

```diff
@@ -69,6 +69,8 @@ public abstract class Player : ScreenWithBeatmapBackground, ISamplePlaybackDisab
 
         public Action RestartRequested;
 
+        public Action OnSeek;
```

Comment:
No xmldoc. Also not marked as an `event` which means anything could overwrite the previous usage.

Reply:
In addition, I'd move this inside `GameplayClockContainer`, as `Player` does not seem like the right place to be handling / binding to seek operations.

Reply:
> No xmldoc. Also not marked as an `event` which means anything could overwrite the previous usage.

Resolved.

> In addition, I'd move this inside `GameplayClockContainer`, as Player does not seem like the right place to be handling / binding to seek operations.

With this change, all of `GameplayClockContainer` is now cached, not just its `GameplayClock`.